OnlineShop/OnlineShop.DAL/Repositories/Implementations/OrderItemRepository.cs

```diff
@@ -1,17 +1,30 @@
-﻿using OnlineShop.DAL.Entities.Implementations;
+﻿using Microsoft.EntityFrameworkCore;
+using OnlineShop.DAL.Entities.Implementations;
+using OnlineShop.DAL.Exceptions;
+using OnlineShop.DAL.Infrastructure;
 using OnlineShop.DAL.Repositories.Interfaces;
+using System.Collections.Immutable;
 
 namespace OnlineShop.DAL.Repositories.Implementations;
 
 public class OrderItemRepository:IBaseRepository<OrderItem>
 {
-    public Task<OrderItem> GetById(Guid id)
+    public ShopDbContext _context;
+
+    public OrderItemRepository(ShopDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<OrderItem> GetById(Guid id)
```

Comment:
Always add the suffix "Async" to the names of methods that have an Async or async modifie